src/Framework/FileMock.php

```diff
@@ -40,16 +48,51 @@ public static function create($content, $extension = NULL)
 	}
 
 
+	public static function register()
+	{
+		if (!self::$registered) {
```

Comment:
Maybe can be replaced with stream_get_wrappers() https://github.com/nette/safe-stream/commit/4b9bb3266a537e59b10a2932f05dca47333420fc